### ISSUE TITLE
tutorial / hints

### ISSUE THREAD

**Issue by spaetzlecode** [ltjax/click_to_repair#10]
consider if more teaching of the user is needed

**Comment by spaetzlecode** [comment]
main problem when i observed new testers:
People don't understand it's "click and hold" not just "click once" or "click really fast". The game title is also misleading.

Maybe the wrench should be displayed permanently as mouse cursor, and only rotate while clicked?

**Comment by ltjax** [comment]
> Maybe the wrench should be displayed permanently as mouse cursor, and only rotate while clicked?

Nah, we tried this during the jam. It does not help and it's really hard to read the cursor position with that. (And it looks shitty)

**Comment by ltjax** [comment]
Maybe we can wait a bit until repairing actually starts, so the players can figure out that you need to hold. This can be further reinforced by somehow "starting" the wrench animation, e.g. slowly fading in the wrench or starting the movement slowly.

**Comment by ltjax** [comment]
Alternatively, we can just remove the penalty for it and treat repeated clicking on the same part exactly the same as holding.
